engines/dfc_provider/app/services/supplied_product_builder.rb

```diff
@@ -49,10 +49,10 @@ def self.import_product(supplied_product)
 
   def self.apply(supplied_product, variant)
     variant.product.assign_attributes(
-      name: supplied_product.name,
       description: supplied_product.description,
```

Comment:
Would this be a problem for description also? 
Or do we need to keep this in order to achieve the current project..

Reply:
> Would this be a problem for description also?

Yes, variants don't have descriptions, only products do. And when you change it through one variant then all get changed. It's a limitation we can't avoid at the moment. Better than no description update at all.